bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/target/TargetParamCodecs.scala

```diff
@@ -0,0 +1,60 @@
+package edu.gemini.spModel.target
+
+import edu.gemini.spModel.core._
+import edu.gemini.spModel.core.Target._
+import edu.gemini.spModel.pio.Param
+import edu.gemini.spModel.pio.codec._
+
+import squants.motion.{ Velocity, KilometersPerSecond }
+
+import scalaz._, Scalaz._
+
+object TargetParamCodecs {
+
+  implicit val RedshiftParamCodec: ParamCodec[Redshift] =
+    ParamCodec[Double].xmap(Redshift(_), _.redshift)
+
+  implicit val AngleParamCodec: ParamCodec[Angle] =
+    ParamCodec[Double].xmap(Angle.fromDegrees, _.toDegrees)
+
+  implicit val ParallaxParamCodec: ParamCodec[Parallax] =
+    ParamCodec[Angle].xmap(Parallax(_), _.angle)
+
+  implicit val VelocityParamCodec: ParamCodec[Velocity] =
+    ParamCodec[Double].xmap(KilometersPerSecond(_), _.toKilometersPerSecond)
+
+  implicit val RadialVelocityParamCodec: ParamCodec[RadialVelocity] =
```

Comment:
This will conflict with PR #581 as `RadialVelocity` is gone but that's a minor detail